src/main/java/edu/ie3/datamodel/models/input/system/type/EvTypeInput.java

```diff
@@ -31,6 +35,8 @@ public class EvTypeInput extends SystemParticipantTypeInput {
    * @param eCons Consumed electric energy per driven distance
    * @param sRated Rated apparent power for this type of EV (typically in kW)
```

Comment:
In my opinion this issue could be solved by change sRated into sRatedAC and at some field for sRatedDC. Thus you added sRatedAC everywhere, the sRated need to be removed everywhere or otherway round :)